src/Controller/ArticlesController.php

```diff
@@ -616,14 +616,24 @@ public function figuresAction(Request $request, string $id, int $version = null)
                     'figures' => $all['figures'],
                     'videos' => $all['videos'],
                     'tables' => $all['tables'],
+                    'data availability' => $all['dataAvailability'],
                     'data sets' => $all['generatedDataSets']->append(...$all['usedDataSets']),
                     'additional files' => $all['additionalFiles'],
                 ], Callback::method('notEmpty'));
             })
             ->then(Callback::mustNotBeEmpty(new NotFoundHttpException('Article version does not contain any figures or data')))
             ->then(function (array $all) {
+                if (1 === count($all) && isset($all['data availability'])) {
+                    return new ViewModel\MessageBar('There are no figures or additional files');
+                }
+
+                unset($all['data availability']);
+
                 return new ViewModel\MessageBar(Humanizer::prettyList(...array_map(function (string $text, Sequence $items) {
                     if (1 === count($items)) {
+                        if ('data availability statements' === $text) {
```

Comment:
where does this string come from? Can't find it in the existing code or this diff

Reply:
Looks left over from https://github.com/elifesciences/journal/pull/1219/commits/b4c1e262fe7e88a20df4809ac9c9885d981f6fea, but is redundant anyway?